backend/db/ddl-12-csets_to_ignore.jinja.sql

```diff
@@ -1,19 +0,0 @@
--- Table: csets_to_ignore ----------------------------------------------------------------------------------------------
```

Comment:
## Delete: `csets_to_ignore`
Deleted the DDL and references to it. It was being recreated during the refresh.